pkg/auth/requests/authenticate.go

```diff
@@ -2,6 +2,8 @@ package requests
 
 import (
 	"context"
+	"github.com/rancher/rancher/pkg/auth/providers"
+	"github.com/sirupsen/logrus"
```

Comment:
Imports should be sorted, built in at the top with an empty line and third party imports below